### PR TITLE
Mention old docs archive in the upgrade doc, don't pause the operator latest versions

### PR DESCRIPTION
**Pull request by fiowro**

### CODE REVIEW

File: docs/update.md [hunk 1]
```diff
@@ -11,38 +11,13 @@ Distribution for PostgreSQL.
...
-
-        The `pgbouncer` user password is stored in encrypted form, and
-        therefore it is not included in the above command. If you know this
-        password and/or would like to update it, please add it as
-        `pgbouncer: base64encodednewpassword` to the resulted Secret manually.
-        Otherwise, this password needs no actions and will be overwritten by the
-        Operator during upgrade.
-
-3. Remove the old Operator and start the new Operator version:
+1. Remove the old Operator version:
```
Comment (hors) [inline docs/update.md:20]: We should not remove old operator. We need to update the operator.
How to do it:
```
curl -s https://raw.githubusercontent.com/percona/percona-postgresql-operator/1.x/deploy/operator.yaml | yq w -d'4' -  "spec.template.spec.containers[0].env[0].value" "update" | kubectl apply -f -
kubectl wait --for=condition=Complete job/pgo-deploy --timeout=90s
```

**Review by hors — CHANGES_REQUESTED** [review]
How to update DB:

remove 1 and 3 steps:
```
1 Pause the cluster in order to stop all possible activities:
3 Now you can enable the `pgbouncer` again:
```
and just use the following command to update cluster

`    $ kubectl patch perconapgcluster/<cluster-name> --type json -p '[{"op": "replace", "path": "/spec/backup/backrestRepoImage", "value": "percona/percona-postgresql-operator:{{ release }}-ppg14-pgbackrest-repo"},{"op":"replace","path":"/spec/backup/image","value":"percona/percona-postgresql-operator:{{ release }}-ppg13-pgbackrest"},{"op":"replace","path":"/spec/pgBadger/image","value":"percona/percona-postgresql-operator:{{ release }}-ppg14-pgbadger"},{"op":"replace","path":"/spec/pgBouncer/image","value":"percona/percona-postgresql-operator:{{ release }}-ppg14-pgbouncer"},{"op":"replace","path":"/spec/pgPrimary/image","value":"percona/percona-postgresql-operator:{{ release }}-ppg14-postgres-ha"},{"op":"replace","path":"/spec/userLabels/pgo-version","value":"v{{ release }}"},{"op":"replace","path":"/metadata/labels/pgo-version","value":"v{{ release }}"}]'`

and you can check that all deployment get new images 
`kubectl get deployment -o 'jsonpath={.spec.template.spec.containers[0].image}{"\n"}' --watch`

File: docs/update.md [hunk 2]
```diff
@@ -11,52 +11,36 @@ Distribution for PostgreSQL.
...
+          value: update
+...
+```
+
+You can automate this with the [yq tool](https://github.com/mikefarah/yq/#install) as
+follows, assuming that you are upgrading to the Operator version {{ release }}:
+
+``` {.bash data-prompt="$" }
+$  curl -s https://raw.githubusercontent.com/percona/percona-postgresql-operator/v{{ release }}/deploy/operator.yaml | | yq w - "spec.template.spec.containers[0].env[3].value" "update" | kubectl apply -f -
+$ kubectl wait --for=condition=Complete job/pgo-deploy --timeout=90s
```
Comment (nmarukovich) [inline docs/update.md:38]: @fiowro did you test the commnad? I see at least problem with` | | `
and we have only one env here https://github.com/percona/percona-postgresql-operator/blob/1.x/deploy/operator.yaml#L173

File: docs/update.md [hunk 3]
```diff
@@ -154,38 +138,17 @@ Semi-automatic update of Percona Distribution for PostgreSQL should be used with
...
 instead.
 
-The following steps will allow you to update the Operator to current version
+The following command will allow you to update the Operator to current version
 (use the name of your cluster instead of the `<cluster-name>` placeholder).
 
+``` {.bash data-prompt="$" }
+$ kubectl patch perconapgcluster/<cluster-name> --type json -p '[{"op": "replace", "path": "/spec/backup/backrestRepoImage", "value": "percona/percona-postgresql-operator:{{ release }}-ppg14-pgbackrest-repo"},{"op":"replace","path":"/spec/backup/image","value":"percona/percona-postgresql-operator:{{ release }}-ppg13-pgbackrest"},{"op":"replace","path":"/spec/pgBadger/image","value":"percona/percona-postgresql-operator:{{ release }}-ppg14-pgbadger"},{"op":"replace","path":"/spec/pgBouncer/image","value":"percona/percona-postgresql-operator:{{ release }}-ppg14-pgbouncer"},{"op":"replace","path":"/spec/pgPrimary/image","value":"percona/percona-postgresql-operator:{{ release }}-ppg14-postgres-ha"},{"op":"replace","path":"/spec/userLabels/pgo-version","value":"v{{ release }}"},{"op":"replace","path":"/metadata/labels/pgo-version","value":"v{{ release }}"}]'
```
Comment (nmarukovich) [inline docs/update.md:145]: Why do we have 
`{"op":"replace","path":"/spec/backup/image","value":"percona/percona-postgresql-operator:{{ release }}-ppg13-pgbackrest"}`

File: docs/update.md [hunk 4]
```diff
@@ -154,38 +138,17 @@ Semi-automatic update of Percona Distribution for PostgreSQL should be used with
...
+!!! note
 
-    ``` {.bash data-prompt="$" }
-    $ kubectl patch perconapgcluster/<cluster-name --type json -p \
-        '[
-            {"op":"replace","path":"/spec/pgBouncer/size","value":1}
-        ]'
-    ```
+    The above example is composed in asumption of using PostgreSQL 14 as
+    a database management system. For PostgreSQL 13 you should change all
```
Comment (nmarukovich) [inline docs/update.md:151]: asumption  -> assumption

**Review by nmarukovich — APPROVED** [review]

**Pull request merged**